src/sidebar/services/test/annotations-exporter-test.js

```diff
@@ -118,36 +120,36 @@ Total annotations: 5
 Total replies: 1
 
 Annotation 1:
-${isoDate}
+${formattedNow}
```

Comment:
I wonder if we should add a `Created: ` prefix here for consistency with other fields. On the other hand, it adds some noise to the output.

Reply:
I considered that as well. On the one hand, a date is probably more straightforward to identify, but on the other hand, the lack of a label makes it harder to know what the date is, so I think it would be beneficial to prefix it with `Created`